docs/topics/async.txt

```diff
@@ -8,15 +8,14 @@ Asynchronous support
 
 Django has support for writing asynchronous ("async") views, along with an
 entirely async-enabled request stack if you are running under
-:doc:`ASGI </howto/deployment/asgi/index>` rather than WSGI. Async views will
-still work under WSGI, but with performance penalties, and without the ability
-to have efficient long-running requests.
+:doc:`ASGI </howto/deployment/asgi/index>`. Async views will still work under
+WSGI, but with performance penalties, and without the ability to have efficient
+long-running requests.
 
-We're still working on asynchronous support for the ORM and other parts of
-Django; you can expect to see these in future releases. For now, you can use
-the :func:`sync_to_async` adapter to interact with normal Django, as well as
-use a whole range of Python asyncio libraries natively. See below for more
-details.
+We're still working on async support for the ORM and other parts of Django.
+You can expect to see this in future releases. For now, you can use the
+:func:`sync_to_async` adapter to interact with the sync parts of Django, as
+well as a whole range of async-native Python libraries.
```

Comment:
Perhaps I'm reading this incorrectly but shouldn't the last line be `whole range of synchronous Python libraries`

Reply:
Indeed!